README.md

```diff
@@ -142,16 +154,61 @@ best_it_contentful:
     type:     annotation
 ```    
 
-Just add the reset controller to your routing (_we suggest to protected it with a http auth password_) and input this
- url in your contentful webhook config and you are ready to go.
+To activate the webhooks in contentful follow the following and guide [guide](https://www.contentful.com/developers/docs/concepts/webhooks/)
+and configure the webhook with the following table:
+
+| Type  | Event   | URL    | Description                |
+|-------|---------|--------|----------------------------|
+| Entry | create  | /fill  | A entry has been created. The cached entry will be created.   |
+| Entry | publish | /fill  | A entry has been published. The cached entry will be overwriten. |
+| Entry | save    | /fill  | A entry has been saved. The cached entry will be overwriten.     |
+| Entry | delete  | /reset | A entry has been deleted. The cached entry will be deleted.  |
+
+The cache warmer is automatically enabled and should run at every call of the following command:
+
+```console
+$ ./bin/console cache:warmup
+```
 
+or 
 
-The most simple cache reset is the direct reset on an id or the array of the ids from an collection (used as cache 
-tags). 
+```console
+$ ./bin/console cache:clear
+```
 
 **If you have cache entries which are not matched to the entry ids directly but to your own custom cache ids, you need to 
 fill the config value caching.collection_consumer with this custom cache ids to reset them anytime another cache is 
 reset.**
+Because of technical limitation the raw entry from contentful is not saved in the cache but a parsed array 
+representation instead. The parsing of a the fetched entry is possible via individual response parsers that convert the
+entry object into an array before it is saved in the cache.
+
+It is possible to implement custom parsers if you need to modify the parsed result. This custom parser can be attached 
+to the fetch operation for the entries. If you have attached a custom parser the parsed array representation for
+this parser will be returned from the cache instead of the default implementation. 
+
+To make this work you need to register your custom parsers to a content type in the configuration. 
+With this configured everytime a entry from contentful with the configured contenttype is saved in the cache,
+the parse result of the custom response parser is also saved in the cache. If you fetch a entry from the cache with a 
+custom parser the correct result for this parser is returned. If you fetch a result without a parser the default parser 
+result is returned.
+
+Add the following configuration to your config file to activate the custom parsers:
+ 
+ ```yaml
+    best_it_contentful:
+        caching:
+            response_parser:
+                # The configured response parser for the content type manufacturer
+                manufacturer:
+                    - 'best_it_contentful.routing.route_collection_response_parser'
```

Comment:
Why is this an array? Can I add multiple parser for one type?

Reply:
Thats right, you can add multiple parsers by type